src/Forms/GridFieldHtmlFragment.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class GridFieldHtmlFragment implements GridField_HTMLProvider
```

Comment:
Could you add a phpdoc for this class please to explain what it does? Also I think the "button" references are misleading